scipy/stats/tests/test_odds_ratio.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_equal, assert_allclose
+from scipy.stats import nchypergeom_fisher, hypergeom
+from scipy.stats._odds_ratio import odds_ratio
+from .fisher_exact_results_from_r import data
+
+
+class TestOddsRatio:
```

Comment:
Input validation tests?

Reply:
I added some input validation tests and tests of the odds ratio and its confidence interval for `kind='sample'`.

Reply:
: )
Would you agree that they are a llittle less thorough than, say, the tests of the conditional odds ratio?
Do you not think they're important?

Reply:
I added more substantial tests (e.g. confidence interval other than the default, one-sided confidence intervals).